GUIDELINES-RELATIONS.md

```diff
@@ -0,0 +1,49 @@
+
+Queste linee guida sono da intendersi come regole _di buon senso_ e di corretta comunicazione.
+
+## Relazioni con altri progetti o community
+
+Il coinvolgimento delle persone è un aspetto fondamentale che sta alla base di questo progetto, di conseguenza intraprendere relazioni con progetti che sposano la nostra stessa filosofia è un aspetto fondamentale.  
+Lo stesso vale per le community o per qualsiasi gruppo formatosi attorno a filosofie simili alla nostra.
+
+## Pubblicazione di contenuti
+
+La pubblicazione di contenuti, che siano essi video, slide, articoli o post sui social, è un aspetto fondamentale per la crescita del progetto, di conseguenza è importante che i contenuti pubblicati siano consoni e che rispettino le linee guida del progetto.
+
+Per quanto possibile, si ricorda di mantenere un linguaggio semplice, gender-neutral e che rispetti il nostro [Codice di Comportamento](https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md).  
+Chiaramente non è possibile controllare ogni singolo contenuto pubblicato o analizzare in tempo reale ogni frase detta durante una discussione, un meetup o una live, ma è importante che ci si ricordi che quando si parla a nome del progetto ogni parola detta è importante e può avere un impatto positivo o negativo sulla percezione del progetto stesso.
+
+## Hashtag
+
+Non avendo ancora stabilito un titolo ufficiale per il progetto, si consiglia di utilizzare gli hashtag `#IlLibroOpenSource` e `#LibroOpenSource` per identificare il progetto, i suoi contenuti e le sue attività.
+
+## Esposizione dei valori del progetto
+
+Il progetto si basa su alcuni valori fondamentali, che sono:
+
+- `Open Source`: Il progetto vive grazie al supporto di tutti, e tutti possono contribuire in qualsiasi modo ritengano opportuno.
+- `Open Governance`: Il progetto è aperto a tutti, non esistono gerarchie o ruoli, e tutti possono contribuire in qualsiasi modo ritengano opportuno. Chiunque può proporre un cambiamento o una modifica, e tutti possono esprimere la propria opinione.
+- `Collaborazione`: Avere la stessa opinione su tutto, in una qualsiasi community, è piuttosto difficile, se non addirittura impossibile. È importante che tutti possano esprimere la propria opinione e che tutti abbiano come obiettivo comune il miglioramento del progetto e la collaborazione reciproca, a testimonianza della nostra convinzione che insieme possiamo creare qualcosa di più grande della somma delle sue parti.
+
+## Esposizione del concetto di Ambassador
+
+Il concetto di Ambassador è stato introdotto per identificare le persone che si impegnano in prima linea nella stesura dei contenuti o di altri aspetti fondamentali del progetto.
+
+Nonostante questo, è importante ricordare che il progetto è aperto a tutti e che non esistono gerarchie o ruoli, e che le persone che ricoprono il ruolo di Ambassador sono più dei _garanti_ delle linee guida e dei valori del progetto che dei _leader_ da seguire.
```

Comment:
@Cadienvan metterei in questo punto anche una descrizione di come si diventa Ambassador, specificando le regole attraverso il sistema di votazione e i momenti in cui si può entrare all'interno dell'organizzazione